apps/smart-contracts/core/test/unit/TokenSender.test.ts

```diff
@@ -269,8 +346,58 @@ describe('=> TokenSender', () => {
       expect(outputToken.transfer).not.called
     })
 
+    it('reverts if sent amount > account limit and caller != recipient', async () => {
+      priceOracle.get.returns(OUTPUT_TOKEN_UNIT) // 1 for 1
+      const outputAmount = await calculateExpectedOutput(TEST_LIMIT.add(1))
+      expect(outputAmount).to.be.eq(TEST_LIMIT.add(1))
```

Comment:
```suggestion
      expect(outputAmount).eq(TEST_LIMIT.add(1))
```
same for rest of file